libs/fluid-elements/tab-group/src/lib/tab/tab.ts

```diff
@@ -99,45 +99,77 @@ export class FluidTab extends LitElement {
     return this._active;
   }
   set active(active: boolean) {
+    const oldActive = this._active;
     // Only set active true if not disabled
     this._active = this.disabled === false ? active : false;
-    // TODO: Figure out why using the old value does not work. The attribute vanishes in the dom/is not set
-    this.requestUpdate('active');
+    this.requestUpdate('active', oldActive);
     this.tabindex = this.active ? 0 : -1;
   }
   private _active = false;
 
-  /** Dispatches the custom event  */
+  /** Defines whether the user focused an element by tabbing or not */
+  @property({ type: Boolean, reflect: false })
+  get tabbed(): boolean {
+    return this._tabbed;
+  }
+  set tabbed(tabbed: boolean) {
+    const oldTabbed = this.tabbed;
+    this._tabbed = tabbed;
+    this.requestUpdate('tabbed', oldTabbed);
+    this.tabindex = tabbed === true ? 0 : -1;
+  }
+  private _tabbed = false;
+
+  /** Dispatches the custom event with the tabid of the clicked tab  */
   private dispatchActiveTabEvent(): void {
     this.dispatchEvent(new FluidTabActivatedEvent(this.tabid));
   }
 
-  /** Handles the click event */
+  /** Handles the click event. Dispatches the tab when a new tab was clicked */
   private handleClick(): void {
     if (!this._active) {
       this.dispatchActiveTabEvent();
     }
   }
 
+  /** Called when the tab loses focus. Removes the focus outline and sets the tab up for TAB navigation. */
+  private handleBlur(): void {
+    this.tabbed = false;
+    this.tabindex = 0;
```

Comment:
The setting of the tabIndex contradicts what happens in the setter of the tabbed. 
The tabbed setter sets the tabindex to -1 if the set value is `false`. 
In line 138 you set it to 0 explicitly. 

I don't think that this is right.